feature-libs/cart/saved-cart/core/store/actions/saved-cart.action.ts

```diff
@@ -121,7 +129,7 @@ export class SaveCart extends StateUtils.EntityLoadAction {
     public payload: {
       userId: string;
       cartId: string;
-      saveCartName?: string;
+      saveCartName: string;
```

Comment:
let's keep this optional as the api as the api does not require to have the save cart name. Even if it was for the wires, we can make the form required, but make our services have the optional param